Lingua-App/Lingua/Lingua/Scenes/Projects/ProjectsView.swift

```diff
@@ -12,6 +12,12 @@ struct ProjectsView: View {
   
   var body: some View {
     NavigationSplitView {
+      Text("")
```

Comment:
I would rather create a view like `📁 Projects` and keep first column collapsed by default, so we don't have that not useful button on the right side

Reply:
fix the toolbar title and subtitle for selected project